doc/articles/features/working-with-accessibility.md

```diff
@@ -97,13 +97,13 @@ When the value of `WinRTFeatureConfiguration.Accessiblity.HighContrast` is chang
 - A child with the same accessible name as its parent is accessibility focusable.
 - `Control` doesn't receive focus when accessibility focused.
 - `TabIndex` is not supported.
```

Comment:
```suggestion
- A child with the same accessible name as its parent is accessibility focusable.
- `Control` doesn't receive focus when accessibility is focused.
- `TabIndex` is not supported.
```

Reply:
I'm not sure about this one. Maybe it meant `accessibility-focused`?

Reply:
Looks like "accessibility focused" is used in multiple places:

https://learn.microsoft.com/en-us/dotnet/api/android.views.accessibility.accessibilitynodeinfo.setrequestinitialaccessibilityfocus?view=net-android-34.0

https://learn.microsoft.com/en-us/dotnet/api/android.views.view.isaccessibilityfocused?view=net-android-34.0

https://learn.microsoft.com/en-us/dotnet/api/android.views.accessibility.accessibilitynodeinfo.accessibilityfocused?view=xamarin-android-sdk-13

https://developer.apple.com/documentation/swiftui/view/accessibilityfocused(_:)

Let's keep as-is for now to get this merged and we will discuss and adjust later if needed.